docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Elytron_OIDC_Client.adoc

```diff
@@ -173,6 +173,148 @@ It is also possible to configure the scope values that would be used via the sco
 
 NOTE: If both the deployment and subsystem configuration is present, then the subsystem configuration will override the deployment values.
 
+== Sending Request Objects as a JWT
+
+NOTE: Sending the request object as a JWT is a preview level feature, meaning the WildFly server's stability level must
+be set to `preview` or lower to enable the ability to add `request` or `request_uri` to authentication requests. The
+stability level must not be changed to community or higher after configuring the attributes related to this feature.
+For more information about the stability levels of the WildFly server, please refer to the
+link:Admin_Guide{outfilesuffix}[Admin Guide], particularly the
+link:Admin_Guide{outfilesuffix}#Feature_stability_levels[Feature stability levels] section.
+
+When securing an application using OpenID Connent, the `request` or `request_uri` parameters enable Authentication
+Requests to be signed and optionally encrypted and sent as a Json Web Token (JWT). The `elytron-oidc-client` subsystem
+allows you to specify if you want to enable this using the `authentication-request-format` attribute.
+
+The `authentication-request-format` attribute has values of type `String` and has 3 allowed values:
+
+* `oauth2`: This sends the authentication request as usual following the oauth2 syntax. Parameters, such as,
+`state` are added to the authentication request directly.
+* `request`: This sends the authentication request as a signed and optionally encrypted JWT, which the OpenID provider
+can verify and/or decrypt. The OpenID provider must support request parameters, which is indicated using the
+`request_parameter_supported` field inside the OpenID metadata.
+* `request-uri`: This also sends the authentication request as a signed and optionally encrypted JWT, but `request_uri`
```

Comment:
should this be request_uri?

Reply:
Just noticed that the description of the attributes here kind of overlaps with the content in the Passing Request... sections. Do you think we could simplify this somehow?